Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const makeDurationHours = tasks.map(elem => elem.duration / 60);
+  const TwoHoursOrBigger = makeDurationHours.filter(elem => elem >= 2);
```

Comment:
One small thing here - if you remember the variables naming conventions document here - https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md, the variable names should be camelCasing. Thus, `twoHoursOrBigger` instead of `TwoHoursOrBigger`. 🙂